iothub/device/src/Common/ExceptionTrace.cs

```diff
@@ -103,50 +103,19 @@ public TException TraceException<TException>(TException exception, TraceEventTyp
                 // Only trace if this is the first time an exception is thrown by this ExceptionTrace/EventSource.
                 exception.Data[_eventSourceName] = _eventSourceName;
 
+#if NET451
                 switch (level)
                 {
                     case TraceEventType.Critical:
                     case TraceEventType.Error:
-#if NET451
-                        Trace.TraceError("An Exception is being thrown: {0}", GetDetailsForThrownException(exception));
-#endif
-                        ////if (MessagingClientEtwProvider.Provider.IsEnabled(
```

Comment:
I presume this is dead code, so I just removed it. @timtay-microsoft, any context to sway this change?

Reply:
No objections here. This class as a whole seems out of place so I'm not sure what the commented out code was for.